src/tools/rustfmt/src/items.rs

```diff
@@ -1947,6 +1947,10 @@ pub(crate) fn rewrite_struct_field(
     if contains_skip(&field.attrs) {
         return Ok(context.snippet(field.span()).to_owned());
```

Comment:
Unrelated, but it's interesting how we don't represent this with `RewriteError::SkipFormatting`...

Reply:
@compiler-errors I think the reason why we don't use `RewriteError::SkipFormatting` here is that returning an `Err` results in later struct fields not being formatted.

Reply:
Well for the purposes of structs w/ default in them, it shouldn't really matter, since the whole struct goes unformatted today (prior to the regression that introduced the bug, of course).